src/components/molecules/ScheduleEvent/ScheduleEvent.tsx

```diff
@@ -39,12 +54,36 @@ export const ScheduleEvent: React.FC<ScheduleEventProps> = ({
   scheduleStartHour,
   personalizedEvent: isPersonalizedEvent = false,
 }) => {
+  const eventRef: RefObject<HTMLDivElement> = useRef(null);
+  const isMinimallyLongEvent = event.duration_minutes >= MIN_LONG_EVENT_LENGTH;
+  const isShortEvent = event.duration_minutes <= SHORT_EVENT_LENGTH;
+  const isEventLong = event.duration_minutes >= LONG_EVENT_LENGTH;
+
   const { userId } = useUser();
+  const {
+    isShown: isExpanded,
+    hide: hideExpanded,
+    show: showExpanded,
+  } = useShowHide();
+  const {
+    isShown: isExpandedStatic,
+    toggle: toggleExpandedStatic,
+    hide: hideExpandedStatic,
+  } = useShowHide();
```

Comment:
Why is there both `isExpanded` and `isExpandedStatic` and what's the nuances/differences between them/why they are both needed?

Reply:
`isExpanded` is triggered upon hovering on an element/toggles off when mouse leaves that component and `isExpandedStatic` is toggled when user explicitly clicks on an icon and stays this way until:
1) user clicks on `collapse` icon
2) user clicks on anything except the event component ( which would open up a modal with event info )
so having two is, at least in my head, required